src/MembershipManager.php

```diff
@@ -189,6 +189,9 @@ public function getGroupIds(EntityInterface $entity, $group_type_id = NULL, $gro
       $query = $this->entityTypeManager
         ->getStorage($target_type)
         ->getQuery()
+        // When rebuilding node_access grants then checking node_access is
```

Comment:
Maybe: `// Disable entity access so fetching the groups related to group content are not affected by the current user.`